docs/quilt/README.md

```diff
@@ -0,0 +1,67 @@
+# Maintainer
+
+Allen Institute for Cell Science ([cells@alleninstitute.org](mailto:cells@alleninstitute.org)) 
+
+# Overview
+
+The Allen Institute for Cell Science is providing the datasets associated with the manuscript “A human induced pluripotent stem (hiPS) cell model for the holistic study of epithelial to mesenchymal transitions (EMTs)” (Hookway et al. 2024, bioRxiv: https://www.biorxiv.org/content/10.1101/2024.08.16.608353v1) 
+
+# Contents & Usage: 
+
+This dataset can be accessed on quilt [here](https://open.quiltdata.com/b/allencell/tree/aics/emt_timelapse_dataset/).
+
+The `emt_timelapse_dataset` directory is organized in the following structure.
+
+<img src="https://github.com/AllenCell/EMT_data_analysis/blob/quilt-readme/docs/quilt/dataset_structure.svg" width="500px" />
+
+## data
+
+The `data` directory contains all the data used in the study “A human induced pluripotent stem (hiPS) cell model for the holistic study of epithelial to mesenchymal transitions (EMTs)”. Each movie has a unique movie ID. For convenience, each specific OME-Zarr file-type associated with a single movie has been assigned a specific filename extension (shown in the Table below). Hence, each file in the `data` directory has a naming convention: movie-id_file-extension.ome.zarr.
+
+File type | Filename Extension 
+----------|-----------------
+Microscope images | raw_converted.ome.zarr 
+All cells mask | all_cells_mask.ome.zarr 
+H2B nuclei segmentation mask | H2B_nuclear_segmentation.ome.zarr 
+EOMES nuclei segmentation mask | EOMES_nuclear_segmentation.ome.zarr 
+CytoGFP ground truth segmentation mask | cytoGFP_ground_truth_segmentation.ome.zarr 
+CollagenIV ground truth segmentation mask | collagenIV_ground_truth_segmentation.ome.zarr 
+CollagenIV segmentation probability | collagenIV_segmentation_probability.ome.zarr 
+CollagenIV segmentaion mesh | collagenIV_segmentation_mesh 
+
+For improved data accessibility we provide further information in the `manifests` section below.
+
+## manifests
+
+The `manifests` directory contains 6 manifests for streamlined data accessibility. Each manifest file and its contents are explained below. 
+
+* imaging_and_segmentation_data.csv: contains both the links to download and links to visualize without download, all the imaging data, segmentation data, and mesh data. This csv also provides columns that contain all accompanying metadata from this study.  
```

Comment:
Can we standardize the capitalization of this CSV names?

Reply:
And can we put the file names between ` `?

Reply:
I think @smishra3 was using the capitalization of the CSVs we uploaded?

Reply:
Added backticks in d68f17d